README.md

```diff
@@ -24,25 +24,28 @@ What is the first thing we need to do before we start styling a web page?
 
 ### Challenge One
 
-- Add a css reset
+- Add CSS Normalize
 
-Before we start styling our web page we need to add a CSS reset. This is because browsers add their own styling to HTML. These styles can clash with our own ones, causing weeiiird behaviour 🙀
+Before we start styling our web page we need to add a CSS reset. This is because a lot of people that love cats are using old browsers that apply inconsistent styles to our cat cards 🙀
 
-Can you figure out how to add a CSS Reset to our project? 
+Can you figure out how to add [CSS Normalize](https://necolas.github.io/normalize.css/) to our project? 
 
 Have a go, if you get stuck read on to find out how...
 
+### How to add a CSS Normalize
```

Comment:
Hey @EllieJudge 

Contrary to the instructions I don't think I would recommend the `npm` approach for using the normalize CSS

Because this particular project isn't a Node project (its just raw HTML/CSS) then I would probably just advise them to copy the contents and put them in their own normalize.css file

If it was a react app (and utilising npm for package management) then using the install approach makes sense I think.

Reading up on it for React it seems they already cater for it if you wish to use it, so I read this material:

https://create-react-app.dev/docs/adding-css-reset/

Which lead me to find that under the hood it uses this Normalize CSS

https://github.com/csstools/normalize.css

And then if you read the instructions for standard HTML it recommends this approach

```
<link href="https://unpkg.com/@csstools/normalize.css" rel="stylesheet" />
```